feature-policy/resources/feature-policy-wakelock.html

```diff
@@ -0,0 +1,9 @@
+<script>
```

Comment:
Can you check the naming for this file? If it's supposed to be used in an iframe, you might want to call it ".sub.html". See other tests that do this. That will help distinguish this file from actual test.

Reply:
[Tests involves multiple origins](http://web-platform-tests.org/writing-tests/server-features.html#tests-involving-multiple-origins) should use ".sub" postfix, this is just a resource file, we can leave it as is.

Reply:
Thanks for clarifying. I think then I screwed up somewhere else 😊